### PR TITLE
Fixes the exception in batch-explain notebook in the official folder

### PR DESCRIPTION
**Pull request by krishr2d2**
Opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb:8]: <p>do not update copyright year</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewferlitsch) [inline notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb:8]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb:8]: <p>Move Dataset after Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (krishr2d2) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Ok, changed it back to 2021.

Comment (krishr2d2) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Changed it.

Comment (krishr2d2) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_batch_explain.ipynb]: Moved the dataset section down, after the Objective section.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**